.github/workflows/molecule-requirements.txt

```diff
@@ -0,0 +1,4 @@
+ansible>=9.1.0,<10.0.0
+ansible-core~=2.16
+molecule>=6.0.2,<7.0.0
+molecule-plugins[podman]>=23.5.0,<24.0.0
```

Comment:
should we be so specific? I think i'd feel better if these weren't pinned. This feels like a great way to only discover things were broken when someone tries to deploy it on newer ansible/etc, when we could've discovered it earlier in molecule testing.

Reply:
I feel divided here because my life is a constant fight against things that break because changes are not sufficiently controlled (e.g. [usegalaxu-eu/issues#499](https://github.com/usegalaxy-eu/issues/issues/499)), meaning that when I am attempting to solve a single problem, I need to solve many at once because the infrastructure has been neglected for a very long time. In normal circumstances, I would be ok with doing what you say, let (noncritical) things break slowly (although I would still prefer pinning).

Pinning stuff like this should at least make everything VERY stable (notice that the actions in the workflow are also pinned and even the runner image).

However, in the long run it will not solve the problem, because nobody values maintenance. The EU will never pay you to go over your playbooks and update things, or extend/fork [dependabot](https://github.com/dependabot) to do it for you.

Again I consider pinning better.